cns/service/main.go

```diff
@@ -646,6 +646,7 @@ func main() {
 	}
 	host, _ := os.Hostname()
 	z = z.With(zap.String("hostname", host), zap.String("version", version), zap.String("kubernetes_apiserver", os.Getenv("KUBERNETES_SERVICE_HOST")))
+	config.Logger = z
```

Comment:
don't pass the bare root logger, create a derived one (`z.WithFields(...)`) that indicates it's the `Service` Logger